zero2w.nix

```diff
@@ -9,6 +9,8 @@
   ];
 
   nixpkgs.hostPlatform = "aarch64-linux";
+  # ! Need a trusted user for deploy-rs.
+  nix.settings.trusted-users = [ "bob" ];
```

Comment:
this can also be the "@wheel"; I was getting `error: cannot add path '/nix/store/...' because it lacks a signature by a trusted key`